content/applications/websites/website/web_design/building_blocks/structure.rst

```diff
@@ -0,0 +1,69 @@
+================
+Structure blocks
+================
+
+The website configurator provides a range of :guilabel:`Structure` :doc:`building blocks <../building_blocks>`
+to design your website's layout, including elements like headings, images, and text.
+
+To add a block to your web page, click :guilabel:`Edit`, select a block from the
+:guilabel:`Structure` section, and drag and drop it to your website.
+
+.. image:: structure/structure-blocks.png
+   :alt: Drag and drop a block from the structure section.
+   :scale: 75%
+
+This documentation presents the :ref:`Banner <structure/banner>` and :ref:`Masonry
+<structure/masonry>` building blocks.
+
+.. _structure/banner:
+
+Banner block
+------------
+
+The :guilabel:`Banner` block combines a title, text, images, and a call to action, making it
+suitable for placement at the top of your website.
+
+A default background image and content are displayed when adding the :guilabel:`Banner` block to
+your web page. They can be edited by going to the :guilabel:`Customize` tab in the website
+configurator.
+
+   .. image:: structure/default-image-content.png
+      :alt: Default background image and content.
+
+Call to action
+~~~~~~~~~~~~~~
+
+The :guilabel:`Call to Action` button encourages visitors to take a specific action, for example, to
+consult your shop, download your eBook, or make an appointment in order to boost your sales.
+
+.. image:: structure/call-to-action.png
+   :alt: The Call to Action button encourages visitors to take a specific action.
+   :scale: 75%
+
+To customize the :guilabel:`Call to Action` button, select it and click the :guilabel:`Edit Link`
+icon. Additional features appear in the :guilabel:`Inline Text` section of the website configurator.
+Add the link to the page you want to redirect to and customize the button according to your needs.
+
+.. image:: structure/inline-text.png
+   :alt: Configure the Call to Action button.
+   :scale: 75%
+
+.. _structure/masonry:
+
+Masonry block
+-------------
+
+The :guilabel:`Masonry` building block offers a range of templates to create a unique website style.
```

Comment:
```suggestion
The :guilabel:`Masonry` building block offers a range of templates to create a unique website style.
```
This doesn't describe what the block is about. You could say something like "The masonry block associates image and text "bricks" in various combinations".
But "to create a unique website style." should be removed (no marketing BS here)